l10n_it_reverse_charge/models/account_rc_type.py

```diff
@@ -26,16 +28,57 @@ class AccountRCTypeTax(models.Model):
     sale_tax_id = fields.Many2one(
         'account.tax',
         string='Sale Tax',
-        required=True)
```

Comment:
Come mai?

Reply:
Perché mi sembrava che nel caso con **With additional supplier self invoice** venisse chiamato solo `generate_supplier_self_invoice`, in chiamata abbiamo chiarito che però viene chiamato anche `generate_self_invoice` in https://github.com/OCA/l10n-italy/blob/500a8918b9e25b546e90bff675c1e69993a89497/l10n_it_reverse_charge/models/account_invoice.py#L443 sull'autofattura passiva generata che insiste sullo stesso tipo di Inversione Contabile, perché ha la stessa posizione fiscale assegnata in https://github.com/OCA/l10n-italy/blob/500a8918b9e25b546e90bff675c1e69993a89497/l10n_it_reverse_charge/models/account_invoice.py#L427.

Ora ho corretto, puoi controllare se va meglio?